modules/core/test/test_mat.cpp

```diff
@@ -2355,5 +2359,15 @@ TEST(Mat, regression_18473)
     EXPECT_EQ((int)5, (int)m.at<short>(19, 49, 99));
 }
 
+TEST(CUDA_Stream, construct_cudaFlags)
+{
+#ifdef HAVE_CUDA
```

Comment:
Guard the whole test (or the file as mentioned above) with `#ifdef HAVE_CUDA`.

IMHO, "#else" branch is not really needed (it is hard enough to introduce such regression).

Reply:
Ok. I'll incorporate with the coming PR update.